{"pipelineKey":"8494b0a6-293e-4cab-9d38-9ef8689299bd","name":"Pipeline_dataflow error","description":"","definition":"{\"workerLimit\":4,\"tasks\":[{\"children\":[\"DataflowSlSn9\"],\"iteratee\":{},\"id\":\"Terminator\",\"type\":\"Terminator\"},{\"parent\":[\"Terminator\"],\"stopOnTaskFailure\":\"N\",\"dfName\":\"Dataflow_Error\",\"stopDependentTasks\":\"N\",\"label\":\"Data\",\"type\":\"runDataFlow\",\"dfKey\":\"3fa98dcd-6c9e-4612-afde-bf707a1ad217\",\"livyServer\":\"\",\"children\":[],\"iteratee\":{},\"name\":\"Dataflow_Error\",\"id\":\"DataflowSlSn9\",\"parameters\":\"limit_rows:limit 1000\"}]}"}